test/bus_test.dart

```diff
@@ -190,6 +199,9 @@ void main() {
         Vector({'a': 0}, {'a_shrunk': 0}),
```

Comment:
you can add more expected output vectors to the same row if you're using the same input.  for example:
```dart
Vector({'a': 0}, {'a_shrunk': 0, 'a_neg_shrunk': 0}),
```